README.md

```diff
@@ -6,20 +6,36 @@ on the [GitHub Applications Page](https://github.com/settings/applications).
 
 ## Basic Usage
 
-    use OmniAuth::Builder do
-      provider :github, ENV['GITHUB_KEY'], ENV['GITHUB_SECRET']
-    end
+Add to your `Gemfile`:
+gem 'omniauth-github'
 
-## Github Enterprise Usage
+Then `bundle install`
+
+Add to an initializer `config/initializers/omniauth.rb`:
+```
+Rails.application.config.middleware.use OmniAuth::Builder do
+  provider :github, ENV['GITHUB_CLIENT_ID'], ENV['GITHUB_CLIENT_SECRET']
+end
+```
+## Github Callback
+
+When a user visits `/auth/github` and are redirected and signin at Github they are redirected to the callback specified in the Github application settings.
```

Comment:
When a user on your site visits `/auth/github` they will be redirected to Github and asked to verify permissions. When they accept, they will be sent back to your site based on the callback setting for the application you setup.

Perhaps a link to some Github documentation on how to add a Oauth application?  

Also may want to put something in here about using 127.0.0.1:3000 vs. localhost:3000 when doing local work with Github Oauth?  (Hint: 127.0.0.1 works, localhost will not.)